src/v3/src/components/PhoneAuthenticator/PhoneAuthenticator.tsx

```diff
@@ -231,81 +176,45 @@ const PhoneAuthenticator: UISchemaElementComponent<UISchemaElementComponentWithV
       <Box
         display="flex"
         flexWrap="wrap"
-        dir="ltr"
+        flexDirection={languageDirection === 'rtl' ? 'row-reverse' : 'row'}
```

Comment:
This is needed because now that we're setting inputs to be LTR via theme overrides on `InputBase`, if we kept the original `dir="ltr"` on this `Box` then it would be trickier to force the labels for the extension and phone number inputs to still be RTL. However, without `dir="rtl"`, the order of the extension and phone number inputs swap so we need this `flexDirection` to make sure that the phone number input is always on the left and the extension is always on the right